docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,49 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM file
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
+```
+
+Upload a DICOM file that is compressed as a `.zip`,`.tar.gz` or `.tgz`.
+$CandID must be the full name of the file and its extension.
+$IsPhantom must be `'Y'` or `'y'` if it's a phantom file, else `'N'` or `'n'` 
+if not.
+
+Returns JSON data upon success having the form:
+
+```js
+{
+    "success":"Uploaded",
+    "mri_upload_id":"3"
+}
+```
+
+mri_upload_id is the Upload ID for the file.
+
+### 5.4 Process DICOM file
+```
+POST /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
+```
+
+Launch processing of an uploaded DICOM file that has not been unprocessed.
+
+The JSON must be of the form:
+
+```js
+{
+    "Filename":$Filename, 
+    "mri_upload_id":$mri_upload_id, 
+    "IsPhantom":$IsPhantom, 
+    "CandID":$CandID,
+    "VisitLabel":$VisitLabel
+}
+```
+
+Returns JSON data with a status if processing has been launched in the form:
+
+```js
+{"success":"Process launched"}
+```
```

Comment:
It should also return a response code of "202 Accepted" (and be documented as such..)